### PR TITLE
Added documentation

### PR DESCRIPTION
**Pull request by afs25**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,56 @@
...
+
+ 
+### API endpoints
+None
+
+## Future direction
+This service predates VIEWS. The first implementation was developed to
+support the now discontinued integration with MyInterview. Incoming files
+from MyInterview were saved in a S3 bucket that is not provisioned by
+this stack. Hence, an AWS lamdba (MonitorIncomingBucket) was
```
Comment (sophiemachin) [inline README.md:42]: spelling, should be 'lambda'

File: README.md [hunk 2]
```diff
@@ -1,2 +1,56 @@
...
+| Amazon S3 Event Notifications | s3-to-sdhs | s3:ObjectCreated:*      | Triggers transfer of audio file to SDHS                     |
+
+ 
+### API endpoints
+None
+
+## Future direction
+This service predates VIEWS. The first implementation was developed to
+support the now discontinued integration with MyInterview. Incoming files
+from MyInterview were saved in a S3 bucket that is not provisioned by
```
Comment (sophiemachin) [inline README.md:41]: 'an S3 bucket'

File: README.md [hunk 3]
```diff
@@ -1,2 +1,56 @@
...
+this stack. Hence, an AWS lamdba (MonitorIncomingBucket) was 
+implemented to scan the incoming bucket regularly for new files.
+And because files arriving from MyInterview did not contain
+any metadata, a "ResearchProjects" table specifying active
+projects and interviewers MyInterview usernames had to be maintained
+in order to identify what project each file belonged to. The
```
Comment (sophiemachin) [inline README.md:47]: Don't need so much detail here

File: README.md [hunk 4]
```diff
@@ -1,2 +1,56 @@
...
+following resources related to the MyInterview integration
+are deprecated and could/should be deleted:
+- Dynamodb "ResearchProjects" table
+- "MockIncomingBucket" S3 bucket
+- MonitorIncomingBucket lambda
+- code and tests related to these resources
+
+It would be good to notify a SNS topic every time a file is
```
Comment (sophiemachin) [inline README.md:55]: an SNS topic

**Review by sophiemachin — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,2 +1,55 @@
...
+project details, a "ResearchProjects" table had to be maintained   
+to lookup what project each file belonged to. The
+following resources related to the MyInterview integration
+are deprecated and could/should be deleted:
+- Dynamodb "ResearchProjects" table
+- "MockIncomingBucket" S3 bucket
+- MonitorIncomingBucket lambda
+- code and tests related to these resources
+
+It would be good to notify an SNS topic every time a file is
```
Comment (this-andy) [inline README.md:54]: I'd prefer if we raised another event.  That allows other processing to be invoked too.  That could include notifying sis topic

**Review by this-andy — COMMENTED** [review]
See comment at end re events/SNS

**Pull request merged**